pkg/commands/process/settings/settings.go

```diff
@@ -40,15 +40,16 @@ type WorkerOptions struct {
 }
 
 type Config struct {
-	Client       *api.API
-	Worker       WorkerOptions      `mapstructure:"worker" json:"worker" yaml:"worker"`
-	Scan         flag.ScanOptions   `mapstructure:"scan" json:"scan" yaml:"scan"`
-	Report       flag.ReportOptions `mapstructure:"report" json:"report" yaml:"report"`
-	Policies     map[string]*Policy `mapstructure:"policies" json:"policies" yaml:"policies"`
-	Target       string             `mapstructure:"target" json:"target" yaml:"target"`
-	Rules        map[string]*Rule   `mapstructure:"rules" json:"rules" yaml:"rules"`
-	BuiltInRules map[string]*Rule   `mapstructure:"built_in_rules" json:"built_in_rules" yaml:"built_in_rules"`
-	CacheUsed    bool               `mapstructure:"cache_used" json:"cache_used" yaml:"cache_used"`
+	Client             *api.API
+	Worker             WorkerOptions      `mapstructure:"worker" json:"worker" yaml:"worker"`
+	Scan               flag.ScanOptions   `mapstructure:"scan" json:"scan" yaml:"scan"`
+	Report             flag.ReportOptions `mapstructure:"report" json:"report" yaml:"report"`
+	Policies           map[string]*Policy `mapstructure:"policies" json:"policies" yaml:"policies"`
+	Target             string             `mapstructure:"target" json:"target" yaml:"target"`
+	Rules              map[string]*Rule   `mapstructure:"rules" json:"rules" yaml:"rules"`
+	BuiltInRules       map[string]*Rule   `mapstructure:"built_in_rules" json:"built_in_rules" yaml:"built_in_rules"`
+	CacheUsed          bool               `mapstructure:"cache_used" json:"cache_used" yaml:"cache_used"`
+	BearerRulesVersion string             `mapstructure:"bearer_rules_version" json:"bearer_rules_version" yaml:"bearer_rules_version"`
```

Comment:
Nervous of making `Config` a hold-all... but I couldn't think of a better place for this to go. I considered adding it to `flag.ScanOptions` or `flag.ReportOptions` but it's not strictly an option so this didn't seem right. Suggestions welcome 🙏

Reply:
I think this is ok for now